routes/mentors.js

```diff
@@ -64,4 +64,27 @@ router.get("/mentors/:id", verifyRoles([ROLE_NAMES.member]),  async (req, res) =
   }
 });
 
+// Update mentor status
+router.patch("/mentors/:id", verifyRoles([ROLE_NAMES.member]), async (req, res) => {
```

Comment:
Please don't forget to update swagger & when you will do it, please mark GET /mentor/:id as done too, because I forgot to update it last time :D
Also, endpoint description in Swagger seems to be inaccurate, it states that "Only users with admin or mentor role can change mentor status." when it's rather "Available to users with elevated role (admin) only or to user owning the card". Please double check everything else is up to date for this endpoint